### PR TITLE
Bumped version to 0.9.0

### PR DESCRIPTION
**Pull request by ekoval**

### CODE REVIEW

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

File: sitewit/__init__.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
 """Python client for the SiteWit API."""
 
-__version__ = '0.8.0'
+__version__ = '0.9.0'
```
Comment (RayeN) [inline sitewit/__init__.py]: patch version bump, since changes are backwards compatible

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

File: sitewit/__init__.py (hunk 1 above)
Comment (ekoval) [inline sitewit/__init__.py]: That's always been a mystery for me how we change versions. All I found for now is:

```
Semantic Versioning[1] is a formal convention for specifying compatibility using a three-part version number: major version; minor version; and patch. The patch number is incremented for minor changes and bug fixes which do not change the software's application programming interface (API). The minor version is incremented for releases which add new, but backward-compatible, API features, and the major version is incremented for API changes which are not backward-compatible. 
```

Comment (RayeN) [inline sitewit/__init__.py]: That applies to libs which aren't under development.  For libs under dev (when major version < 1) the rules are different.  And we came into agreement that to be able to distinguish between backwards compatible and non-compatible changes we should increment minor for 1st case and patch for the second.

Comment (ekoval) [inline sitewit/__init__.py]: Do you have any proofs which explain this? Is it some internal agreement?

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

File: sitewit/__init__.py (hunk 1 above)
Comment (RayeN) [inline sitewit/__init__.py]: The same document `Semantic Versioning` says about that.  And regarding our agreement - I'm not sure if we document it anywhere.  But it sounds like a common sense rule.

Comment (ekoval) [inline sitewit/__init__.py]: > The same document Semantic Versioning says about that. 

Can't see this.

Comment (RayeN) [inline sitewit/__init__.py]:
```
Major version zero (0.y.z) is for initial development. Anything may change at any time. The public API should not be considered stable.
```

Comment (ekoval) [inline sitewit/__init__.py]: Yes, but this says nothing about patch and minor version.

Comment (RayeN) [inline sitewit/__init__.py]: It says it can be changed in any way :)

Comment (ekoval) [inline sitewit/__init__.py]: So, this is some internal agreement.

**Pull request merged**